hackathon_site/registration/forms.py

```diff
@@ -171,6 +173,7 @@ def save(self, commit=True):
 
         self.instance.user = self.user
         self.instance.team = team
+        self.instance.phone_number = re.sub("[^0-9]", "", self.instance.phone_number)
```

Comment:
Needs a unit test, you can put it in the sign up form tests

Reply:
I think it makes more sense to put it in the application form tests no?

Reply:
Whoops, yes you're right!